test-interfaces/src/main/java/org/terracotta/passthrough/IServerInfo.java

```diff
@@ -0,0 +1,19 @@
+package org.terracotta.passthrough;
+
+/**
+ * Provides a server information like configured server port and group port etc
+ *
+ * @author vmad
+ */
+public interface IServerInfo {
```

Comment:
I wonder if we need some other way to identify the specific server, in here.  Ideally, it would be something you could correspondingly query from an entity to build the association.

If our current usage is only in asking for the server and group ports of either an arbitrary server, or all servers (but never a _specific_ server), then what you have here is good.